tests/test_security.py

```diff
@@ -0,0 +1,104 @@
+"""
+https://github.com/ray-project/kuberay/blob/master/docs/guidance/pod-security.md
+Test for pod-security.md in CI
+"""
+import os
+import logging
+import unittest
+import yaml
+from kubernetes import client
+from kubernetes.client.rest import ApiException
+
+from framework.prototype import (
+    RayClusterAddCREvent
+)
+
+from framework.utils import (
+    shell_subprocess_run,
+    CONST,
+    K8S_CLUSTER_MANAGER,
+    OperatorManager
+)
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(
+    format='%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
+    datefmt='%Y-%m-%d:%H:%M:%S',
+    level=logging.INFO
+)
+
+class PodSecurityTestCase(unittest.TestCase):
+    """
+    https://github.com/ray-project/kuberay/blob/master/docs/guidance/pod-security.md
+    Test for the document for the Pod security standard in CI
+    """
+    def test_pod_security(self):
```

Comment:
This looks good.

If you want, you can slightly improve readability by splitting up the test logic's steps
(creating the kind cluster, installing the operator, creating a RayCluster, installing a non-compliant pod)
into sub-functions.
Some of the logic could potentially go into a setupTest method.

Reply:
Thank you for the feedback! I will open a new pr to split up the test logic's steps. For setupTest, I am not sure what kind of test cases will be added to test_security.py in the future.  Do you mind leaving the setupTest method to the time when we need to add a test to test_security.py?

Reply:
Sounds good.